tests/kernel/ib_tests.pm

```diff
@@ -39,18 +39,16 @@ sub check_dmesg {
 }
 
 sub ibtest_slave {
-    my $host1 = get_required_var('IBTEST_IP1');
-    my $host2 = get_required_var('IBTEST_IP2');
+    my $master = get_required_var('IBTEST_IP1');
```

Comment:
how do you know IPTEST_IP1 will be always for master? The scheduler gives the tests one of the 2 workers randomly. So, sometimes master will be IBTEST_IP1 and some other times IBTEST_IP2.

Maybe for your case it is not relevant which IP you get for master, but what I am doing in NFV is a quick trick, checking my own ip with "ip a" command and then calculating the other IP. A bit hacky but it works:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/nfv/prepare_env.pm#L31